ProjectCoimbra.UWP/Project.Coimbra/Pages/OptionsPage.xaml

```diff
@@ -0,0 +1,40 @@
+<!--
+Licensed under the MIT License.
+-->
+<Page
+    x:Class="Coimbra.Pages.OptionsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="#F5F5F5">
+
+	<Grid>
```

Comment:
Mix of tabs and spaces

Reply:
Sorry VS messed that up. Fixing for all files now.